src/QSymbolicsBase/objects.jl

```diff
@@ -0,0 +1,71 @@
+"""This file defines quantum objects (kets, bras, and operators) with various properties"""
+struct SKet <: Symbolic{AbstractKet}
+    name::Symbol
+    basis::Basis
+end
+struct SBra <: Symbolic{AbstractBra}
+    name::Symbol
+    basis::Basis
+end
+struct SOperator <: Symbolic{AbstractOperator}
+    name::Symbol
+    basis::Basis
+end
+const SymQ = Union{SKet, SBra, SOperator}
+isexpr(::SymQ) = false
+metadata(::SymQ) = nothing
+basis(x::SymQ) = x.basis
+
+symbollabel(x::SymQ) = x.name
+Base.show(io::IO, x::SKet) = print(io, "|$(symbollabel(x))⟩")
+Base.show(io::IO, x::SBra) = print(io, "⟨$(symbollabel(x))|")
+Base.show(io::IO, x::SOperator) = print(io, "$(symbollabel(x))")
+Base.show(io::IO, x::SymQObj) = print(io, symbollabel(x)) # fallback that probably is not great
+
+ishermitian(x::SOperator) = false
+isunitary(x::SOperator) = false
+iscommutative(x::SOperator) = false
+
+"""Inverse Operator"""
```

Comment:
what is an "inverse operator"

Reply:
Probably needs a docstring that can answer that questions on its own as well.

Reply:
Sorry, it's a terse docstring, I'll fix that. I defined `SInverseOperator` so one could have the inverse of a symbolic operator with the `inverse` function.